titanic.ipynb

```diff
@@ -43,9 +43,6 @@
         "# Load the Titanic dataset\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 Line 3_
```python
import seaborn as sns
import matplotlib.pyplot as plt
import pandas as pd
```
`pandas` should be removed as it's no longer being used

_Respond and view the context [here](https://gitnotebooks.com/smith-kyle/notebook-diff-tests/pull/26)._

Reply:
rest